docs/developers/staking_backend/staking_backend.mdx

```diff
@@ -55,12 +55,9 @@ properly set up:
 {/* Database Migration (Optional) - Clone Phase-1 database snapshot, apply snapshot to new MongoDB clusters.
 Required only if supporting Phase-1 registration data */}
 
-> **Note:** If upgrading an existing deployment, ensure you gracefully shut down
-> any legacy indexer services before proceeding.
+## Migrating from phase-1 stack
```

Comment:
The below does not offer much information. What does the upgrade process involve, how is it to be performed? Needs more details.